.buildkite/libbeat/pipeline.libbeat.yml

```
@@ -98,11 +99,12 @@ steps:
           automatic:
             - limit: 1
         agents:
-          provider: "gcp"
-          image: "${IMAGE_UBUNTU_X86_64}"
-          machineType: "${GCP_DEFAULT_MACHINE_TYPE}"
+          provider: "aws"
+          image: "${IMAGE_UBUNTU_X86_64_FIPS}"
+          instanceType: "m5.2xlarge"
```

@elastic/observablt-ci I'm currently reusing the AWS images we have available, but we may want to have dedicated images for beats. The (aws) images are missing python3 and may be missing other tools; also it's on aws and the rest of the beats steps seem to prefer gcp

---

To my knowledge, the AMI core image we rely on is provided by another team and is only accessible through AWS. If we choose to use AWS instead of GCP, do you anticipate any potential challenges while using the AWS instead of GCP?

Additionally, could you please share the list of required tools?

@pazone, I would appreciate your assistance in creating the image for Beats. Thank you!

---

https://github.com/elastic/beats/pull/45286

---

Much better now, after I recreated a new VM image, it [installed](https://buildkite.com/elastic/beats-libbeat/builds/17854#0197f467-48e4-4dbe-8db4-3aa768a195bd/170) the microsoft/golang and ran the tests, but they [failed](https://buildkite.com/elastic/beats-libbeat/builds/17854#0197f467-48e4-4dbe-8db4-3aa768a195bd/566-567)

```
panic: crypto/hkdf: use of hash functions other than SHA-2 or SHA-3 is not allowed in FIPS 140-only mode
```

---

If possible, let's use a smaller instance type (4 CPUs and 16 GB of RAM for UTs might be enough?)
```suggestion
          provider: "aws"
          image: "${IMAGE_UBUNTU_X86_64_FIPS}"
          instanceType: "m5.xlarge"
```